app/views/games/index.html.erb

```diff
@@ -1,11 +1,22 @@
 <div class="booyah-box col-xs-10 col-xs-offset-1">
-<h1 class="text-center">List of Games</h1>
+<h3 class="text-center"><%= current_user.email %></h3>
+<% if @games.present? %>
+	<h1 class="text-center">Available Games</h1>
+<% end %>

```

Comment:
You could extend this conditional down the whole page and add an else that would allow for a message like `No games currently available to join.  Create a new one?`.  The last sentence would be a link to the `new` game page.